src/Controller/AvailabilityController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Service\AvailableCarService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+#[Route('/availabilities')]
+class AvailabilityController extends AbstractController
+{
+    #[Route('', methods: ['GET'])]
+    public function listAvailable(Request $request, AvailableCarService $availableCarService): JsonResponse
+    {
+        $stationId = $request->query->get('station_id');
+        $from = $request->query->get('from');
+        $to = $request->query->get('to');
+
+        if (!$this->validateRequest($stationId, $from, $to)) {
+            return new JsonResponse(['error' => 'Invalid or missing parameters'], JsonResponse::HTTP_BAD_REQUEST);
```

Comment:
```suggestion
            return new JsonResponse(['error' => 'Invalid or missing parameters'], JsonResponse::HTTP_OK);
```
To check the candidates context awareness

Reply:
Great point! How about removing the status code instead?

Reply:
Sounds good!